modules/install-upgrade/partials/hardware-sizing.adoc

```diff
@@ -0,0 +1,68 @@
+[cols=4*,options=header]
```

Comment:
Some numbers are not the same as in the latest "Pulsar Cluster Deployment Reference Architecture and Recommended Production Settings" document. Please stop using the numbers in the previously shared customer specific document.
The numbers in the new doc are more aligned with our AS setup.